main.py

```diff
@@ -16,64 +16,68 @@
         {"name": "viewport", "content": "width=device-width, initial-scale=1.0"}
     ],
 )
-initial_days = 7
-
-import constants
 
-df =  constants.NYC_BIKE_API_LINK
+days = 30
+df = constants.NYC_BIKE_API_LINK
```

Comment:
Happy you are using the constants, but I would import the variables you need from the constants. It looks cleaner.

```python
from .constants import (
    NYC_BIKE_API_LINK,
    ...
)
```